client/src/Components/Work.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import '../Style/Work.css'
+import WorkBackground from '../Images/bright.JPEG'
+var sectionStyle = {
+    backgroundImage: `url(${WorkBackground})`,
+    backgroundPosition: 'center',
+    backgroundSize: 'cover',
+    backgroundRepeat: 'y',
+    width: '100%',
+    height: '100%',
+    minHeight:"95vh"
+};
+export default function Work(props) {
+    return (
+        <div style={sectionStyle} id="work-body">
```

Comment:
This is a unique way of doing your css! I like it. Just remember that doing it this way is the same as doing inline CSS so it'll overwrite all css from your imported css where you use it